### PR TITLE
Inputs autofill

### LINKED ISSUE #34: Auto-complete electricity bill details
When a user selects a location and month, auto-complete the KWH in the from depending on the location and month selected.

### PR DESCRIPTION
**Pull request by Rolo123y**
- Auto-filling of KWH input field based on selected MONTH OF BILL
- Auto-filling of TOTAL AMOUNT input field based on if user has entered a value in the USAGE input and if there is a value in the KWH field
- If auto-filled, a yellow question mark image appears on the right of the table which displays a caution message when you hover over it
- some alignment fixes

### CODE REVIEW

**Comment by Kyle199910** [comment]
Hey Ruhollah,

Great job on the changes to the input form. I like how you now have to click the button in order to get the tier and time of use to show up on the screen, it definitely makes the begin button more relevant now, which is great.( Now my repositioning of the button is useless lol). The auto fill in is great and definitely makes it easier to fill out the input form.

However there are a few issues i noticed when testing out the input form. For starters, i think the math for the total calculation is wrong
Capture
Capture2
.
Look sketch to me, but i could be wrong. 

A minor thing: when i was testing the hover for the yellow icon, it took like a second or 2 for the text to show up, which then made me as a user question if the icon was broken, so ideally it would be good if the text showed up instantly when you hover over the icon. Also, besides the color, the icons are the same, which could make users think to click the icon instead of hovering as intended, so ideally a different icon would be bette
Capture3
r

Overall, great job, the input form looks great. Keep up the good work for this last week.

Cheers,
Kyle

File: app/routes.py [hunk 1]
```diff
@@ -149,6 +149,65 @@ def renderInputs2():
...
+
+
+@app.route('/get_autofill_inputTotal', methods=['GET', 'POST'])
+def get_autofill_inputTotal():
+    dataGet = request.get_json(force=True)
+    usage = dataGet['usage']
+    kwh = dataGet['kwh']
+
+    if is_float(usage) == True and is_float(kwh) == True:
+        dataReply = {'total': float(usage) + float(kwh)}
```
Comment (Kyle199910) [inline app/routes.py]: i think should be multiply, not add

Comment (Rolo123y) [reply app/routes.py]: fixed

**Review by Kyle199910 — DISMISSED** [review]
as stated in my feedback, main change is to adjust the math, cause i doubt that 50 khw * $0.08/KWH is $50.08 

also the minor chance would be good to do to avoid comfusion between the red question icon and the yellow question icon

**Comment by Kyle199910** [comment]
the icon and math checks out, looks good. Also for the input form, would it be possible to make the button( time of use or tiered look pressed), something to indicate that time of use or tiered is selected( besides the form)? If not, its fine

**Comment by Kyle199910** [comment]
the new tooltips looks good, much smoother than the old one

**Comment by Rolo123y** [comment]
> the icon and math checks out, looks good. Also for the input form, would it be possible to make the button( time of use or tiered look pressed), something to indicate that time of use or tiered is selected( besides the form)? If not, its fine

done

**Comment by Kyle199910** [comment]
Everything looks and functions well at least from my end: this can be merged

**Review by gaurav613 — COMMENTED** [review]
A few comments:
- Looks like the options to choose bill type don't show up without clicking 'Begin' - is this on purpose, or a bug by any chance
- This is more of a design suggestion: would it be possible to have the 'Choose bill type' above the buttons? It's just to make it look centralized - you can also move the red question mark above the buttons.
billtype

**Comment by gaurav613** [comment]
Is this feature only applicable to Time of Use inputs - because there's no autofill happening in the tiered section?

**Comment by Kyle199910** [comment]
Hey Gaurav, for the Looks like the options to choose bill type don't show up without clicking 'Begin' , i feel that's done on purpose so that the begin button has more relevance, as for the autofill, its works from my end, you just got to select a month earlier than march 2022.

**Comment by gaurav613** [comment]
Looks good, merging now - design changes can be made in another PR.

**Pull request merged**